docs/book/quick-start.md

```diff
@@ -64,11 +64,10 @@ Usage involves instantiating the hydrator, and then passing information to it.
 use Zend\Hydrator;
 $hydrator = new Hydrator\ArraySerializable();
 
-$object = new ArrayObject(array());
```

Comment:
What's wrong with this line?

Reply:
I thought it was an unnecessary line.

It's confusing that the hydrator both mutates its argument and returns the mutated argument.  

The way the doc was makes it look like $object is significant, but really any empty ArrayObject will do.  That's why I changed it, and used the (currently believed) clearer syntax of assigning the return value to $object.

Perhaps the original author meant to emphasize how the hyrdrator merges the old data with the new data, but the example didn't include that.

(And, wow, I have a typo in my PR.  ('an').  Let me fix that...)